.travis/install-aom.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+set -ex
+
+AOM_VERSION="v1.0.0-errata1"
+
+if [ ! -d "$BUILD_DIR/aom-$AOM_VERSION" ]; then
+  # Remove any old versions that might exist from the cache
+  rm -rf "$BUILD_DIR/aom*"
+
+  mkdir -p "$BUILD_DIR/aom"
+  git clone --depth 1 -b v1.0.0-errata1 https://aomedia.googlesource.com/aom aom-$AOM_VERSION
```

Comment:
I'd quote everything which includes variables, could you please change that here and everywhere else (like cd and other scripts similarly)